src/app/utils/ServiceUtils.ts

```diff
@@ -57,6 +57,12 @@ const ServiceUtils = {
 		});
 	},
 
+	getRoleId(roleManager: RoleManager, roleName: string): Role {
```

Comment:
Since it returns the Role object, I suggest renaming to `getRole` for clarity.
```suggestion
	getRole(roleManager: RoleManager, roleName: string): Role {
```